### ISSUE TITLE
Second screen unable to see PlayActor w/ Homebridge running in Docker

### ISSUE THREAD

**Issue by bjoernmayer** [kopiro/homebridge-playstation#30]
**Describe Your Problem:**
I am running homebridge in a Docker container. The second screen app cannot see any device called `PlayActor` and hence I cannot continue with the setup.

**Logs:**

```
Nothing of importance
```

**Plugin Config:**

```json
Nothing of importance as I fail to setup the plugin
```

**Environment:**

* **Plugin Version**: 1.4.1
* **Homebridge Version**: 1.4.0
* **Node.js Version**: 16.13.2
* **NPM Version**: 8.1.2
* **Operating System**: Docker

**Comment by mesoblaste** [comment]
I was having the same problem.
I found a way to go over it :
I installed ps4-waker

```
sudo npm i ps4-waker -g
```

And then connect my PS4 with 2nd screen app using this in the terminal :

```
ps4-waker
```

Which register the credentials in homebridge.

Then when you log in homebridge-playstation, with :

```
homebridge-playstation-login
```

It works.

**Comment by bjoernmayer** [comment]
@mesoblaste worked like a charm.
Do I have to redo these steps after I pull a newer docker image version?

**Comment by mesoblaste** [comment]
I think so, I think it just save the credentials in a file, using ps4-waker, that it also used by homebridge-playstation. I can't say why the process doesn't work directly with homebridge-playstation with the PS4, the process is different for the PS5 but works fine. So far all my PlayStations appears and respond good in Apple home app and that is all I need, thanks to the developers :)

**Comment by bjoernmayer** [comment]
I played around with it some more.
I deleted both credential files of ps4-waker (`~/.ps4-wake.credentials.json`) and playactor (`~/.config/playactor/credentials.json`).

With `playactor login` no device appears in second screen app.
With `ps4-waker` a `PS4-Waker` device appears and I am able to connect. While `ps4-waker` already exits, the second-screen app continues to try to connect to this fake device.
At the same time I once again executed `playactor login`, which then immediately  succeeds.
